pom.xml

```diff
@@ -103,6 +110,18 @@
     </dependency>
     <!-- end of webjars -->
 
+    <!-- Testing -->
```

Comment:
You wouldn't need these with Spring Boot 2.2. Maybe we should make this a 2.2 change?

The exclusion above is unnecessary then as well.

Reply:
Fine, it will simply the XML configuration.
If you want, to remember it, I can create an issue in prevision of the Spring Boot 2.2 upgrade.

Reply:
Thanks. That would be great.